alex_w/server.js

```diff
@@ -0,0 +1,109 @@
+var express = require("express");
+var path = require("path");
+var app = express();
+var mongoose = require('mongoose');
+var bodyParser = require('body-parser');
+
+mongoose.connect('mongodb://localhost/mongoose_dashboard');
+
+var PandaSchema = new mongoose.Schema({
+ name: String,
+ age: Number
+})
+mongoose.model('Panda', PandaSchema); 
+var Panda = mongoose.model('Panda') 
+
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(express.static(path.join(__dirname, "./static")));
+app.set('views', path.join(__dirname, './views'));
+app.set('view engine', 'ejs');
+
+//
+//      GET ROUTES
+//
+
+app.get('/', function(req, res) {
+    Panda.find({}, function(err, pandas) {
+    if(err) {
+      console.log('something went wrong');
```

Comment:
Ideally, send errors to the client. If you're just debugging, also console log the 'err' object that our callback gives us, it will have some interesting details inside!